### ISSUE TITLE
Improving the startup user experience

### ISSUE THREAD

**Issue by jphalip** [JesusFreke/fusionIdea#23]
I understand the need to set up some type of key validation since the debugging session is initiated from outside of Fusion 360. However, it can be just a tiny bit tedious to have to manually copy-paste the key. As a compromise, what do you think about just displaying a message box that says something like "A debugging session with key XYZ was started. Do you want to proceed?". Then the user would just need to click OK. This would be a similar workflow to, say, a bluetooth pairing scenario.

What do you think?

**Comment by JesusFreke** [comment]
Yeah, if the key is shown, then the user can still verify it's from the expected source at least. I'm not opposed, although I don't actually find it too annoying, especially since you only have to do it once (at least, until IDEA or Fusion is restarted)
